spk/sonarr/Makefile

```diff
@@ -1,6 +1,6 @@
 SPK_NAME = nzbdrone
 SPK_VERS = $(shell date +%Y%m%d)
```

Comment:
I would really rename package too and set its upstream version...
I guess it should be possible to automate transition, copying `nzbdrone/target/var` to new `sonarr/target/var` and triggering `nzbdrone` package removal.
@maxrogers May you please investigate such package transition? It would be really helpful for other applications too.

Reply:
Can we trigger an official package removal from within the script? Such that it's also removed from the the package manager?

Reply:
I think so. `synopkg uninstall` but must be tested in case it cannot be triggered when installation is already in progress. https://forum.synology.com/enu/viewtopic.php?t=144513

Reply:
https://github.com/SynoCommunity/spksrc/pull/3803#issuecomment-564166738